doc/api/n-api.md

```diff
@@ -1697,8 +1697,9 @@ napi_status napi_get_typedarray_info(napi_env env,
 - `[in] typedarray`: `napi_value` representing the `TypedArray` whose
 properties to query.
 - `[out] type`: Scalar datatype of the elements within the `TypedArray`.
-- `[out] length`: `Number` of elements in the `TypedArray`.
-- `[out] data`: The data buffer underlying the typed array.
+- `[out] length`: number of elements in the `TypedArray`.
```

Comment:
As all other descriptions begin with upper case, maybe we can just remove backticks around `Number` or add `The` before `number` as in the next line?